src/API/Site/Controllers/MerchantCenter/ShippingRateBatchController.php

```diff
@@ -85,6 +112,39 @@ protected function get_batch_create_callback(): callable {
 		};
 	}
 
+	/**
+	 * Get the callback for creating items via batch.
+	 *
+	 * @return callable
+	 */
+	protected function get_batch_suggestions_callback(): callable {
+		return function( Request $request ) {
+			$country_codes = $request->get_param( 'country_codes' );
+
+			$responses = [];
+			$errors    = [];
+			foreach ( $country_codes as $country_code ) {
+				$new_request = new Request( 'GET', "/{$this->get_namespace()}/{$this->route_base}/suggestions" );
+				$new_request->set_param( 'country_code', $country_code );
+
+				$response = $this->server->dispatch_request( $new_request );
+				if ( 200 !== $response->get_status() ) {
+					$errors[] = $response->get_data();
+				} else {
+					$responses[] = $response->get_data();
+				}
+			}
+
+			return new Response(
+				[
+					'errors'  => $errors,
+					'success' => $responses,
+				],
+				201
```

Comment:
Why 201 and not 200?

Reply:
Yeah, that would be the correct status code. Fixed it.